src/main/java/gyro/google/compute/AbstractBackendServiceResource.java

```diff
@@ -204,12 +204,12 @@ public void setProtocol(String protocol) {
     /**
      * The resource URL for the security policy associated with this backend service.
      */
-    @Output
-    public String getSecurityPolicy() {
+    @Updatable
```

Comment:
The docs should be updated.
1. Should not mention resource url, rather security policy
2. Should also mention that can be added only if cdn  is disabled.